packages/admin/src/PrismaTable/Form/Inputs.tsx

```diff
@@ -227,6 +227,13 @@ const defaultInputs: Omit<FormInputs, 'Upload'> = {
       });
     }
 
+    React.useEffect(() => {
+      if (data && data[`findOne${field.type}`]) {
+        setState(data[`findOne${field.type}`]);
+        setValue(field.name, data[`findOne${field.type}`]);
+      }
+    }, [data]);
```

Comment:
you can't send `data` as dependencies in `useEffect `  because you can't compare to objects this will make it update every render.

Reply:
i see. i will close this pull.